tests/robo/upload-logs.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
This file is similar to `tests/nightly/upload.sh` except that the log artifact names and storage buckets are different.